gramps/plugins/drawreport/ancestortree.py

```diff
@@ -5,6 +5,7 @@
 # Copyright (C) 2010       Jakim Friant
 # Copyright (C) 2014       Paul Franklin
 # Copyright (C) 2010-2015  Craig J. Anderson
+# Copyright (C) 2014       Dave Khuon - option to color the box by gender
```

Comment:
>  - option to color the box by gender
Comment

You don't need to add that as git blame will show you did it, please follow how others have added themselves.